From fe15ea8b86f19bf7c4d1a4ea7452f1bb04b8933a Mon Sep 17 00:00:00 2001
From: Armando Uribe <x0095078@ti.com>
Date: Thu, 22 Jul 2010 20:25:31 -0500
Subject: [PATCH 065/116] staging: tidspbridge: Remove unused magic number

This patch removes an unused magic number from the cod_manager
structure.

Signed-off-by: Armando Uribe <x0095078@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/tidspbridge/pmgr/cod.c |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/drivers/staging/tidspbridge/pmgr/cod.c b/drivers/staging/tidspbridge/pmgr/cod.c
index 0cbe6e5..81102d2 100644
--- a/drivers/staging/tidspbridge/pmgr/cod.c
+++ b/drivers/staging/tidspbridge/pmgr/cod.c
@@ -43,9 +43,6 @@
 /*  ----------------------------------- This */
 #include <dspbridge/cod.h>
 
-/* magic number for handle validation */
-#define MAGIC	 0xc001beef
-
 /*
  *  ======== cod_manager ========
  */
@@ -58,7 +55,6 @@ struct cod_manager {
 	struct dbll_fxns fxns;
 	struct dbll_attrs attrs;
 	char sz_zl_file[COD_MAXPATHLENGTH];
-	u32 ul_magic;
 };
 
 /*
@@ -234,8 +230,6 @@ int cod_create(struct cod_manager **mgr, char *str_zl_file,
 	if (mgr_new == NULL)
 		return -ENOMEM;
 
-	mgr_new->ul_magic = MAGIC;
-
 	/* Set up loader functions */
 	mgr_new->fxns = ldr_fxns;
 
@@ -295,7 +289,6 @@ void cod_delete(struct cod_manager *cod_mgr_obj)
 		cod_mgr_obj->fxns.delete_fxn(cod_mgr_obj->target);
 		cod_mgr_obj->fxns.exit_fxn();
 	}
-	cod_mgr_obj->ul_magic = ~MAGIC;
 	kfree(cod_mgr_obj);
 }
 
-- 
1.7.1

